packages/core/src/application/modal/createApplication.modal.controller.js

```diff
@@ -71,11 +76,12 @@ module(CORE_APPLICATION_MODAL_CREATEAPPLICATION_MODAL_CONTROLLER, [
     let navigateTimeout = null;
 
     const routeToApplication = () => {
-      navigateTimeout = $timeout(() => {
-        $state.go('home.applications.application', {
-          application: this.application.name,
-        });
-      }, 1000);
+      $uibModalInstance.close(this.application);
+      // navigateTimeout = $timeout(() => {
+      //   $state.go('home.applications.application', {
+      //     application: this.application.name,
+      //   });
+      // }, 1000);
```

Comment:
Can this be deleted?

Reply:
Ah yes, thanks for catching. I was testing moving it out to the modal result handlers but forgot to remove here.